src/app-layout/visual-refresh-toolbar/drawer/styles.scss

```diff
@@ -20,6 +34,30 @@
   pointer-events: auto;
   word-wrap: break-word;
 
+  &.last-opened {
+    @include mobile-only {
+      z-index: constants.$drawer-z-index-mobile;
+    }
+  }
+
+  &.drawer-global {
+    @include desktop-only {
+      border-inline-start: 1px solid awsui.$color-border-layout;
+    }
+
+    @include mobile-only {
+      display: none;
+
+      &.last-opened {
+        display: block;
+      }
+    }
+  }
+
+  &.drawer-hidden {
```

Comment:
**Just comment:**

There is a possible CSS conflict. `.drawer.drawer-global.last-opened` will override `.drawer.drawer-hidden` (more class names, more specific).

I would recommend moving more code into JS to make it easier to control and write tests (you cannot test for the computed `display` value, but you can assert `drawer-hidden` or `drawer-visible` class)